src/service/server/grpcServer.go

```diff
@@ -11,11 +11,13 @@ import (
 )
 
 var (
-	PolicyChan  chan tp.KubeArmorHostPolicyEventWithIdentity
-	ClusterIp   string
-	Clusterport string
-	podIp       string
-	EtcdClient  *etcd.EtcdClient
+	PolicyChan           chan tp.KubeArmorHostPolicyEventWithIdentity
+	ClusterIp            string
+	Clusterport          string
+	podIp                string
+	EtcdClient           *etcd.EtcdClient
+	IdentityToStreamMap  map[uint16]pb.KVM_SendPolicyServer
+	IdentityToConnStatus map[uint16]bool
```

Comment:
Instead of maintaining two maps, we can reduce it to one..
 
```
type PolicyServerConn struct {
    stream *pb.KVM_SendPolicyServer
    status  bool
}

var (
    ...
    ConnCache map[uint16]PolicyServerConn
    ...
)
```

Reply:
@wazir-ahmed I already tried the same once, but faced some issue. Will try once and update.